README.md

```diff
@@ -99,33 +99,36 @@ Contact: firstname.lastname@hpi.de
 
 ## Contributors
 
--   Yannick   Bäumer
--   Lawrence  Benson
--   Timo      Djürken
--   Fabian    Dumke
--   Moritz    Eyssen
--   Martin    Fischer
--   Christian Flach
--   Pedro     Flemming
--   Johannes  Frohnhofen
--   Adrian    Holfter
--   Sven      Ihde
--   Michael   Janke
--   Max       Jendruk
--   Marvin    Keller
--   Sven      Lehmann
--   Jan       Mattfeld
--   Arne      Mayer
--   Torben    Meyer
--   Leander   Neiß
--   Hendrik   Rätz
--   Johannes  Schneider
--   David     Schumann
--   Arthur    Silber
--   Daniel    Stolpe
--   Jonathan  Striebel
--   Nils      Thamm
--   Carsten   Walther
--   Lukas     Wenzel
--   Fabian    Wiebe
--   Tim       Zimmermann
+-	Yannick   Bäumer
```

Comment:
We changed this to single space on purpose because people were having difficulties with maintaing the formatting.